packages/esm-appointments-app/src/appointments/forms/create-edit-form/appointments-form.component.tsx

```diff
@@ -163,17 +159,7 @@ const AppointmentForm: React.FC<AppointmentFormProps> = ({ appointment, patientU
         </div>
       )}
 
-      {/*   <div className={styles.inputContainer} id="appointment-place">
-        <p>{t('selectAppointmentLocation', 'Select where the appointment will take place')}</p>
-        <ContentSwitcher className={styles.inputContainer} data-testid="appointment-place">
-          <Switch value="facility" id="facility" text={t('facility', 'Facility')}>
-            {t('facility', 'Facility')}
-          </Switch>
-          <Switch value="community" id="community" text={t('community', 'Community')}>
-            {t('community', 'Community')}
-          </Switch>
-        </ContentSwitcher>
-      </div>*/}
+      <p>{t('selectAppointmentLocation', 'Select where the appointment will take place')}</p>
```

Comment:
Added this heading back in, and then went ahead and removed the community/facility selector entirely.